src/middlewares/validators/general.validation.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-useless-escape */
```

Comment:
I think you can resolve this by hitting `ctrl + s` if you have autoformat on save in your editor. Instead of disabling eslint rules

Reply:
I worked on it.